odl/tomo/data/clintec_spect.py

```diff
@@ -0,0 +1,120 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Helpers for reading CLINTEC SPECT data formats.
+
+For the bilinear relationship of linear attenuation coefficients and
+Houndsfield units see, for example
+
+Investigation of the relationship between linear attenuation
+coefficients and CT Houndsfield units using radionuclides for SPECT
+S. Brown et al, Applied Radiation and Isotopes 66 (2008) 1206-1212
+
+and references therein.
+"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+import dicom
+import os
+from natsort import natsorted
+
+
+def read_clintec_CT_reconstruction(path):
+    """Reads the data in Dicom format and returns a HU CT volume.
+
+    Parameters
+    ----------
+    path : a `path` to the folder where Dicom data is
+
+    """
+
+    lstFilesDCM = []  # create an empty list
+    for dirName, subdirList, fileList in os.walk(path):
+        for filename in natsorted(fileList):
+            if ".dcm" in filename.lower():
+                # check whether the file ends by .dcm
+                lstFilesDCM.append(os.path.join(dirName, filename))
+            else:
+                lstFilesDCM.append(os.path.join(dirName, filename))
+
+    dataset = dicom.read_file(lstFilesDCM[0])
+
+    # Load dimensions based on the number of rows, columns, and slices
+    shape = (int(dataset.Rows), int(dataset.Columns), len(lstFilesDCM))
+    data_array = np.zeros(shape, dtype=dataset.pixel_array.dtype)
+    # loop through all the DICOM files
+    for filenameDCM in lstFilesDCM:
+        # read the file
+        ds = dicom.read_file(filenameDCM)
+        # store the raw image data
+        data_array[:, :, lstFilesDCM.index(filenameDCM)] = ds.pixel_array
+
+    rescale_intercept = int(dataset.RescaleIntercept)
+    rescale_slope = int(dataset.RescaleSlope)
+
+    ct_volume = rescale_slope * np.float32(data_array) + rescale_intercept
+
+    return ct_volume, dataset
+
+
+def linear_attenuation_map_from_HU_data(path, a, b):
+    """Converts HU data to linear attenuation map.
+
+    Parameters
+    ----------
+    path : a `path` to the data folder
+    a, b : `tuple`
+        a = (a1, a2) where a1 `float` and a2 `float` are the intercepts
+            of the bilinear curve
+        b = (b1, b2) where b1 `float` and b2 `float` are the slopes
+            of the bilinear curve
```

Comment:
Point to the (to be created) Notes section, otherwise this is just jargon for users.